kiwi-desc-saltboot/saltboot/suse-SLES12/config.xml

```diff
@@ -155,6 +155,9 @@
         <file name="net/packet/*"/>
         <file name="net/sunrpc/*"/>
     </drivers>
+    <drivers> 
+        <file name="drivers/block/virtio_blk.ko" /> 
+    </drivers>
```

Comment:
This should go up to the default profile section. There are already other block drivers.